keepassxc-browser/css/autocomplete.css

```diff
@@ -2,6 +2,11 @@
     display: none;
     z-index: 2147483646;
     position: absolute !important;
+    background-color: #ddd;
+    border: 1px solid rgba(0,0,0,.125);
```

Comment:
I just noticed that this shouldn't be here. Now it adds an extra border to the whole Autocomplete Menu instead of the footer. My bad for not seeing it earlier.